server/src/main/scala/actors/daily/FlightUpdatesSupervisor.scala

```diff
@@ -82,6 +82,7 @@ class FlightUpdatesSupervisor(now: () => SDateLike,
       val terminalDays = terminalDaysForPeriod(fromMillis, toMillis)
 
       terminalsAndDaysUpdatesSource(terminalDays, sinceMillis)
+        .log(getClass.getName)
```

Comment:
do we want to keep this log in?

Reply:
Yes. I discovered it by accident. Without it we don't get any logs when things go wrong in the stream, which is presumably why sometimes the app crashes without a trace in the logs